### PR TITLE
Updating doxygen

### PR DESCRIPTION
**Pull request by PatV3-0**
Updating all the doxygen

### CODE REVIEW

**Comment by hayley-d** [comment]
Failed memcheck - please make sure no code is changed.

**Comment by PatV3-0** [comment]
Will have to sit and actively compare, but will do - possibly been done in earlier headers where the pointers may be getting changed

File: headers/Government.h
Comment (hayley-d) [inline headers/Government.h:1]: Should not have deleted here the point of singleton is to remove possibility of creating another instance that's why I privated copy constructor and assignment operator.

Comment (Kc-Nell) [reply headers/Government.h:1]: I see thanks for letting me know, i must have misread.

**Review by hayley-d — APPROVED** [review]

**Pull request merged**